gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=3.7.0-SNAPSHOT
+projectVersion=3.6.3-SNAPSHOT
```

Comment:
this is a bug fix. lets keep 3.6.3-SNAPSHOT

Reply:
I set it __*Back*__ to 3.6.3?

Reply:
never mind. I am not thinking straight.I changed the PR base to 3.6.x